app/subscription_webhook.py

```diff
@@ -31,3 +33,6 @@ def execute_subscription_webhook(user: User):
             )
     except RequestException as e:
         LOG.error(f"Subscription request exception: {e}")
+
+    event = UserPlanChange(plan_end_time=sl_subscription_end)
```

Comment:
There are many routes that create subscriptions. ManualSub, AppleSub, Coinbase... I would put it in the create of each model so to avoid losing events. Except in PartnerSubscription

Reply:
This is called for every subscription, so it will be invoked regardless of where it comes from

Reply:
This method is used regardless of which subscription is used, so it will be called for all of them :+1: